build.gradle

```diff
@@ -1508,6 +1508,18 @@ project(':streams:upgrade-system-tests-24') {
   }
 }
 
+project(':streams:upgrade-system-tests-25') {
```

Comment:
I realized this just now. We really should test upgrading from 2.5.0 to 2.5.1, and similar for every bugfix release. In other words, we should add/update "LATEST_X_Y" to the test matrix of the X.Y branch after each release.